src/steps/step.go

```diff
@@ -33,5 +34,10 @@ type SerializedRunState struct {
 }
 
 func getRunResultFilename(command string) string {
-	return fmt.Sprintf("/tmp/%s_%s", command, strings.Replace(git.GetRootDirectory(), "/", "_", -1))
+	replaceCharacterRegexp, err := regexp.Compile("[^A-Za-z0-9]")
```

Comment:
It would need to be `[[:^alpha:]]`. And you had originally said alpha-numeric which would be `[[:^alnum:]]`. Do you want to stick with alpha numeric?